x-pack/plugins/reporting/public/notifier/job_failure.tsx

```diff
@@ -30,16 +32,12 @@ export const getFailureToast = (
     ),
     text: toMountPoint(
       <>
-        <EuiCallOut
-          size="m"
-          title={i18n.translate('xpack.reporting.publicNotifier.error.calloutTitle', {
-            defaultMessage: 'The reporting job failed',
-          })}
-          color="danger"
-          iconType="alert"
-          data-test-errorText={errorText}
-        >
-          {errorText}
+        <EuiCallOut size="m" color="danger" data-test-errorText={errorText}>
+          {job.errorCode === errors.VisualReportingSoftDisabledError.code
```

Comment:
Is it possible to move this logic to `ReportingAPIClient.getError(jobId: string): Promise<string>`, where the passed-in errorText comes from?